.github/workflows/assignproj.yml

```diff
@@ -7,14 +7,22 @@ on:
 env:
   MY_GITHUB_TOKEN: ${{ secrets.DOTNET_GITHUB_TOKEN }}
 
+permissions:
+  contents: read
+
 jobs:
   assign_one_project:
     runs-on: ubuntu-latest
     name: Assign to One Project
     steps:
+    - name: Harden Runner
+      uses: step-security/harden-runner@03bee3930647ebbf994244c21ddbc0d4933aab4f # v2.3.0
+      with:
+        egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
```

Comment:
Could you create a ticket to address the TODOs?

Reply:
Done. #1515

Reply:
Thanks!